bldsys/cmake/template/gradle/app.build.gradle.in

```diff
@@ -2,13 +2,14 @@ plugins {
 	id 'com.android.application'
 }
 
-ext.vvl_version='1.3.275.0'
+ext.vvl_version='1.3.296.0'
 apply from: "./download_vvl.gradle"
 
 android {
 	compileSdk 33
-	buildToolsVersion "33.0.0"
-	ndkVersion "25.2.9519653"
+	buildToolsVersion '34.0.0'
+	ndkVersion '27.0.12077973'
+	namespace "com.khronos.vulkan_samples"
 
 	defaultConfig {
 		applicationId "com.khronos.vulkan_samples"
```

Comment:
targetSdk should be set to 35

Reply:
Github won't let me comment where it should be updated; but we can probably bump JAVA_VERSION from 1.8 up to VERSION_26

Reply:
See comment above.

Reply:
Without having to muck about with gradle's wrapper, I bumped my suggested change to VERSION_21.  That should last us another 5 years ;-)

Reply:
Merged your PR, so this should be addressed.